src/Misc/externals.sh

```diff
@@ -87,4 +88,50 @@ function acquireNode ()
     echo Done
 }
 
+function acquireTee ()
```

Comment:
We need to figure out when to accept the eula (cli).  Of course can't be during build.  Likely during config.  We should print the url to the url and ask Y/N whether they accept the EULA during config.  If no, err/fail.  Else exec the cli.  See my script in node agent.  The EULA url is in the vso-agent - we should checkin here and reference.  I'll add an aka.ms url.